README.md

```diff
@@ -144,3 +144,115 @@ To configure joypads, use the built-in menu or the `retroarch-joyconfig` command
 ## Compiling and installing
 
 Instructions for compiling and installing RetroArch can be found in the [Libretro/RetroArch Documentation Center](https://docs.libretro.com/).
+
+## CRT SwitchRes
+
+CRT Switch res will turn on, on the fly. However, you will need to restart retroarch to disable it. With CRT SwitchRes enable Retroarch will start in 2560 @ 480.  
+
+Before enabling the CRT SwitchRes options pleasae make sure you have installed CRTEmudriver and installed some modelines. The minimun modelins for all games to switch correctly are:
+
+- 2560 x 192 @ 60.000000
+- 2560 x 200 @ 60.000000
+- 2560 x 240 @ 60.000000
+- 2560 x 224 @ 60.000000
+- 2560 x 237 @ 60.000000
+- 2560 x 256 @ 50.000000
+- 2560 x 254 @ 55.000000
+- 2560 x 448 @ 60.000000
+- 2560 x 480 @ 60.000000
+
+Install these modelines replacing 2560 with your desired super resuloution. The above resolutions are NTSC only so if you would be playing any PAL content please add pal modelines:
+
+- 2560 x 192 @ 50.000000
+- 2560 x 200 @ 50.000000
+- 2560 x 240 @ 50.000000
+- 2560 x 224 @ 50.000000
+- 2560 x 288 @ 50.000000
+- 2560 x 237 @ 50.000000
+- 2560 x 254 @ 55.000000
+- 2560 x 448 @ 50.000000
+- 2560 x 480 @ 50.000000
+
+Some games will require heigher PAL resolutions which should also be installed:
+
+- 2560 x 512 @ 50.000000
+- 2560 x 576 @ 50.000000
+
+Idealy install all these modelines and everything will work great.
+
+## Super Resolutions
+
+The default super resolution is 2560. It is displayed just under the CRT switch option, which can be found in video settings. This can be changed within the retroarch.cfg. The only compatible resolutions are 1920, 2560 and 3840. Any other resolutions will be ignored and native swithing will be activated.
```

Comment:
Super resolutions are high resolutions on the width, x scale. They are designed so that most if not all width resolutions divide to an exact integer. This makes it be scales of the fly to fit within this resolution without artifacts.

Should this be explained in the Read me?